fava/static/package.json

```diff
@@ -30,6 +30,7 @@
     "jquery-dragster": "github:catmanjan/jquery-dragster",
     "jquery-query-object": "^2.2.3",
     "jquery-stupid-table": "github:medsabir/Stupid-Table-Plugin#feat-npm-publish",
-    "mousetrap": "^1.5.3"
+    "mousetrap": "^1.5.3",
+    "query-string": "^4.1.0"
```

Comment:
Even the author of `jquery-query-object` [doesn't recommend](https://github.com/alrusdi/jquery-plugin-query-object#disclaimer--) using it, so I'm for keeping `query-string` instead (or considering [URI.js](https://github.com/medialize/URI.js) if we need something more powerful)